depext.ml

```diff
@@ -41,11 +41,11 @@ let string_split char str =
   aux 0
 
 let has_command c =
-  let cmd = Printf.sprintf "/bin/sh -c command -v %s" c in
+  let cmd = Printf.sprintf "command -v %s" c in
```

Comment:
I'm not sure this will work well. Based on experience with cmdliner [that's](https://github.com/dbuenzli/cmdliner/blob/master/src/cmdliner.ml#L349-L355) a correct and multiplatform way of checking whether a command exists or not.